walletdb/bdb/db.go

```diff
@@ -68,7 +68,7 @@ func (tx *transaction) ReadWriteBucket(key []byte) walletdb.ReadWriteBucket {
 }
 
 func (tx *transaction) CreateTopLevelBucket(key []byte) (walletdb.ReadWriteBucket, error) {
-	boltBucket, err := tx.boltTx.CreateBucket(key)
+	boltBucket, err := tx.boltTx.CreateBucketIfNotExists(key)
```

Comment:
is this change relevant to the pr?

Reply:
Discussed offline that without this change, there isn't a way to conditionally obtain/create a top-level bucket.